src/admin/class-sign-in-with-google-admin.php

```diff
@@ -364,6 +373,17 @@ public function siwg_show_on_login() {
 
 	}
 
+	/**
+	 * Callback function for Show Google Signup Button on Login Form
```

Comment:
This is actually the callback to display the field in the admin.

Reply:
my mistake, probably I had copy-pasted it and forgot to edit.